xds/build.gradle

```diff
@@ -47,11 +47,13 @@ dependencies {
     annotationProcessor libraries.autovalue
     compileOnly libraries.javax_annotation,
             // At runtime use the epoll included in grpc-netty-shaded
-            libraries.netty_epoll
```

Comment:
Both of these in this file... actually serve no purpose since b118a590 and could be deleted. But you could also have them environment-specific if you didn't want to delete them.

Reply:
I left those files unchanged as the only change i needed was to get the right arch binary for native epoll.